PN: python3
PV: 3.8.14
PR: r0
PKGV: 3.8.14
PKGR: r0
LICENSE: PSF-2.0 & BSD-0-Clause
DESCRIPTION: Python is a programming language that lets you work more quickly and integrate your systems more effectively.
SUMMARY_python3-2to3: Python automated Python 2 to 3 code translator
RDEPENDS_python3-2to3: python3-core
SECTION: devel/python
PKG_python3-2to3: python3-2to3
FILES_python3-2to3:  /usr/bin/2to3* /usr/lib/python3.8/lib2to3
FILES_INFO: {"/usr/bin/2to3": 8, "/usr/bin/2to3-3.8": 95, "/usr/lib/python3.8/lib2to3/Grammar.txt": 8676, "/usr/lib/python3.8/lib2to3/Grammar3.8.14.final.0.pickle": 15309, "/usr/lib/python3.8/lib2to3/PatternGrammar.txt": 793, "/usr/lib/python3.8/lib2to3/PatternGrammar3.8.14.final.0.pickle": 1225, "/usr/lib/python3.8/lib2to3/__init__.py": 7, "/usr/lib/python3.8/lib2to3/__main__.py": 67, "/usr/lib/python3.8/lib2to3/__pycache__/__init__.cpython-38.opt-1.pyc": 129, "/usr/lib/python3.8/lib2to3/__pycache__/__init__.cpython-38.opt-2.pyc": 129, "/usr/lib/python3.8/lib2to3/__pycache__/__init__.cpython-38.pyc": 129, "/usr/lib/python3.8/lib2to3/__pycache__/__main__.cpython-38.opt-1.pyc": 213, "/usr/lib/python3.8/lib2to3/__pycache__/__main__.cpython-38.opt-2.pyc": 213, "/usr/lib/python3.8/lib2to3/__pycache__/__main__.cpython-38.pyc": 213, "/usr/lib/python3.8/lib2to3/__pycache__/btm_matcher.cpython-38.opt-1.pyc": 4877, "/usr/lib/python3.8/lib2to3/__pycache__/btm_matcher.cpython-38.opt-2.pyc": 3306, "/usr/lib/python3.8/lib2to3/__pycache__/btm_matcher.cpython-38.pyc": 4877, "/usr/lib/python3.8/lib2to3/__pycache__/btm_utils.cpython-38.opt-1.pyc": 6150, "/usr/lib/python3.8/lib2to3/__pycache__/btm_utils.cpython-38.opt-2.pyc": 4638, "/usr/lib/python3.8/lib2to3/__pycache__/btm_utils.cpython-38.pyc": 6150, "/usr/lib/python3.8/lib2to3/__pycache__/fixer_base.cpython-38.opt-1.pyc": 6253, "/usr/lib/python3.8/lib2to3/__pycache__/fixer_base.cpython-38.opt-2.pyc": 3502, "/usr/lib/python3.8/lib2to3/__pycache__/fixer_base.cpython-38.pyc": 6253, "/usr/lib/python3.8/lib2to3/__pycache__/fixer_util.cpython-38.opt-1.pyc": 12198, "/usr/lib/python3.8/lib2to3/__pycache__/fixer_util.cpython-38.opt-2.pyc": 10034, "/usr/lib/python3.8/lib2to3/__pycache__/fixer_util.cpython-38.pyc": 12198, "/usr/lib/python3.8/lib2to3/__pycache__/main.cpython-38.opt-1.pyc": 8574, "/usr/lib/python3.8/lib2to3/__pycache__/main.cpython-38.opt-2.pyc": 6821, "/usr/lib/python3.8/lib2to3/__pycache__/main.cpython-38.pyc": 8608, "/usr/lib/python3.8/lib2to3/__pycache__/patcomp.cpython-38.opt-1.pyc": 5397, "/usr/lib/python3.8/lib2to3/__pycache__/patcomp.cpython-38.opt-2.pyc": 4889, "/usr/lib/python3.8/lib2to3/__pycache__/patcomp.cpython-38.pyc": 5639, "/usr/lib/python3.8/lib2to3/__pycache__/pygram.cpython-38.opt-1.pyc": 1260, "/usr/lib/python3.8/lib2to3/__pycache__/pygram.cpython-38.opt-2.pyc": 1055, "/usr/lib/python3.8/lib2to3/__pycache__/pygram.cpython-38.pyc": 1260, "/usr/lib/python3.8/lib2to3/__pycache__/pytree.cpython-38.opt-1.pyc": 23658, "/usr/lib/python3.8/lib2to3/__pycache__/pytree.cpython-38.opt-2.pyc": 14955, "/usr/lib/python3.8/lib2to3/__pycache__/pytree.cpython-38.pyc": 24342, "/usr/lib/python3.8/lib2to3/__pycache__/refactor.cpython-38.opt-1.pyc": 20407, "/usr/lib/python3.8/lib2to3/__pycache__/refactor.cpython-38.opt-2.pyc": 16361, "/usr/lib/python3.8/lib2to3/__pycache__/refactor.cpython-38.pyc": 20440, "/usr/lib/python3.8/lib2to3/btm_matcher.py": 6623, "/usr/lib/python3.8/lib2to3/btm_utils.py": 9966, "/usr/lib/python3.8/lib2to3/fixer_base.py": 6690, "/usr/lib/python3.8/lib2to3/fixer_util.py": 15207, "/usr/lib/python3.8/lib2to3/fixes/__init__.py": 47, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/__init__.cpython-38.opt-1.pyc": 135, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/__init__.cpython-38.opt-2.pyc": 135, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/__init__.cpython-38.pyc": 135, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_apply.cpython-38.opt-1.pyc": 1647, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_apply.cpython-38.opt-2.pyc": 1557, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_apply.cpython-38.pyc": 1673, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_asserts.cpython-38.opt-1.pyc": 1272, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_asserts.cpython-38.opt-2.pyc": 1202, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_asserts.cpython-38.pyc": 1272, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_basestring.cpython-38.opt-1.pyc": 656, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_basestring.cpython-38.opt-2.pyc": 611, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_basestring.cpython-38.pyc": 656, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_buffer.cpython-38.opt-1.pyc": 801, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_buffer.cpython-38.opt-2.pyc": 732, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_buffer.cpython-38.pyc": 801, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_dict.cpython-38.opt-1.pyc": 3259, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_dict.cpython-38.opt-2.pyc": 2365, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_dict.cpython-38.pyc": 3328, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_except.cpython-38.opt-1.pyc": 2811, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_except.cpython-38.opt-2.pyc": 2365, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_except.cpython-38.pyc": 2811, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_exec.cpython-38.opt-1.pyc": 1116, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_exec.cpython-38.opt-2.pyc": 950, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_exec.cpython-38.pyc": 1142, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_execfile.cpython-38.opt-1.pyc": 1664, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_execfile.cpython-38.opt-2.pyc": 1536, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_execfile.cpython-38.pyc": 1690, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_exitfunc.cpython-38.opt-1.pyc": 2299, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_exitfunc.cpython-38.opt-2.pyc": 2227, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_exitfunc.cpython-38.pyc": 2299, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_filter.cpython-38.opt-1.pyc": 2437, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_filter.cpython-38.opt-2.pyc": 1977, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_filter.cpython-38.pyc": 2437, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_funcattrs.cpython-38.opt-1.pyc": 969, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_funcattrs.cpython-38.opt-2.pyc": 901, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_funcattrs.cpython-38.pyc": 969, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_future.cpython-38.opt-1.pyc": 777, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_future.cpython-38.opt-2.pyc": 674, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_future.cpython-38.pyc": 777, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_getcwdu.cpython-38.opt-1.pyc": 781, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_getcwdu.cpython-38.opt-2.pyc": 715, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_getcwdu.cpython-38.pyc": 781, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_has_key.cpython-38.opt-1.pyc": 2904, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_has_key.cpython-38.opt-2.pyc": 2334, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_has_key.cpython-38.pyc": 2932, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_idioms.cpython-38.opt-1.pyc": 3838, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_idioms.cpython-38.opt-2.pyc": 3342, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_idioms.cpython-38.pyc": 3914, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_import.cpython-38.opt-1.pyc": 2780, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_import.cpython-38.opt-2.pyc": 2469, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_import.cpython-38.pyc": 2780, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_imports.cpython-38.opt-1.pyc": 4388, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_imports.cpython-38.opt-2.pyc": 4324, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_imports.cpython-38.pyc": 4388, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_imports2.cpython-38.opt-1.pyc": 539, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_imports2.cpython-38.opt-2.pyc": 438, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_imports2.cpython-38.pyc": 539, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_input.cpython-38.opt-1.pyc": 943, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_input.cpython-38.opt-2.pyc": 874, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_input.cpython-38.pyc": 943, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_intern.cpython-38.opt-1.pyc": 1127, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_intern.cpython-38.opt-2.pyc": 1063, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_intern.cpython-38.pyc": 1127, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_isinstance.cpython-38.opt-1.pyc": 1544, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_isinstance.cpython-38.opt-2.pyc": 1224, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_isinstance.cpython-38.pyc": 1544, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_itertools.cpython-38.opt-1.pyc": 1543, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_itertools.cpython-38.opt-2.pyc": 1183, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_itertools.cpython-38.pyc": 1543, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_itertools_imports.cpython-38.opt-1.pyc": 1531, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_itertools_imports.cpython-38.opt-2.pyc": 1449, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_itertools_imports.cpython-38.pyc": 1571, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_long.cpython-38.opt-1.pyc": 698, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_long.cpython-38.opt-2.pyc": 634, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_long.cpython-38.pyc": 698, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_map.cpython-38.opt-1.pyc": 3090, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_map.cpython-38.opt-2.pyc": 2294, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_map.cpython-38.pyc": 3090, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_metaclass.cpython-38.opt-1.pyc": 5299, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_metaclass.cpython-38.opt-2.pyc": 3919, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_metaclass.cpython-38.pyc": 5340, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_methodattrs.cpython-38.opt-1.pyc": 931, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_methodattrs.cpython-38.opt-2.pyc": 855, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_methodattrs.cpython-38.pyc": 931, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_ne.cpython-38.opt-1.pyc": 804, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_ne.cpython-38.opt-2.pyc": 759, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_ne.cpython-38.pyc": 804, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_next.cpython-38.opt-1.pyc": 3048, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_next.cpython-38.opt-2.pyc": 2985, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_next.cpython-38.pyc": 3076, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_nonzero.cpython-38.opt-1.pyc": 916, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_nonzero.cpython-38.opt-2.pyc": 857, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_nonzero.cpython-38.pyc": 916, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_numliterals.cpython-38.opt-1.pyc": 1016, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_numliterals.cpython-38.opt-2.pyc": 954, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_numliterals.cpython-38.pyc": 1016, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_operator.cpython-38.opt-1.pyc": 4213, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_operator.cpython-38.opt-2.pyc": 3729, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_operator.cpython-38.pyc": 4213, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_paren.cpython-38.opt-1.pyc": 1383, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_paren.cpython-38.opt-2.pyc": 1249, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_paren.cpython-38.pyc": 1383, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_print.cpython-38.opt-1.pyc": 2257, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_print.cpython-38.opt-2.pyc": 1970, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_print.cpython-38.pyc": 2336, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_raise.cpython-38.opt-1.pyc": 2246, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_raise.cpython-38.opt-2.pyc": 1623, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_raise.cpython-38.pyc": 2246, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_raw_input.cpython-38.opt-1.pyc": 788, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_raw_input.cpython-38.opt-2.pyc": 721, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_raw_input.cpython-38.pyc": 788, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_reduce.cpython-38.opt-1.pyc": 1121, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_reduce.cpython-38.opt-2.pyc": 991, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_reduce.cpython-38.pyc": 1121, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_reload.cpython-38.opt-1.pyc": 1139, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_reload.cpython-38.opt-2.pyc": 1069, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_reload.cpython-38.pyc": 1139, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_renames.cpython-38.opt-1.pyc": 2000, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_renames.cpython-38.opt-2.pyc": 1920, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_renames.cpython-38.pyc": 2000, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_repr.cpython-38.opt-1.pyc": 838, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_repr.cpython-38.opt-2.pyc": 774, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_repr.cpython-38.pyc": 838, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_set_literal.cpython-38.opt-1.pyc": 1676, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_set_literal.cpython-38.opt-2.pyc": 1601, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_set_literal.cpython-38.pyc": 1676, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_standarderror.cpython-38.opt-1.pyc": 713, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_standarderror.cpython-38.opt-2.pyc": 659, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_standarderror.cpython-38.pyc": 713, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_sys_exc.cpython-38.opt-1.pyc": 1404, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_sys_exc.cpython-38.opt-2.pyc": 1235, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_sys_exc.cpython-38.pyc": 1404, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_throw.cpython-38.opt-1.pyc": 1801, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_throw.cpython-38.opt-2.pyc": 1570, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_throw.cpython-38.pyc": 1801, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_tuple_params.cpython-38.opt-1.pyc": 4584, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_tuple_params.cpython-38.opt-2.pyc": 4250, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_tuple_params.cpython-38.pyc": 4584, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_types.cpython-38.opt-1.pyc": 1830, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_types.cpython-38.opt-2.pyc": 1356, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_types.cpython-38.pyc": 1830, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_unicode.cpython-38.opt-1.pyc": 1542, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_unicode.cpython-38.opt-2.pyc": 1363, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_unicode.cpython-38.pyc": 1542, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_urllib.cpython-38.opt-1.pyc": 5992, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_urllib.cpython-38.opt-2.pyc": 5434, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_urllib.cpython-38.pyc": 5992, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_ws_comma.cpython-38.opt-1.pyc": 1112, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_ws_comma.cpython-38.opt-2.pyc": 926, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_ws_comma.cpython-38.pyc": 1112, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_xrange.cpython-38.opt-1.pyc": 2522, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_xrange.cpython-38.opt-2.pyc": 2458, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_xrange.cpython-38.pyc": 2522, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_xreadlines.cpython-38.opt-1.pyc": 1116, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_xreadlines.cpython-38.opt-2.pyc": 987, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_xreadlines.cpython-38.pyc": 1116, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_zip.cpython-38.opt-1.pyc": 1576, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_zip.cpython-38.opt-2.pyc": 1245, "/usr/lib/python3.8/lib2to3/fixes/__pycache__/fix_zip.cpython-38.pyc": 1576, "/usr/lib/python3.8/lib2to3/fixes/fix_apply.py": 2346, "/usr/lib/python3.8/lib2to3/fixes/fix_asserts.py": 984, "/usr/lib/python3.8/lib2to3/fixes/fix_basestring.py": 320, "/usr/lib/python3.8/lib2to3/fixes/fix_buffer.py": 590, "/usr/lib/python3.8/lib2to3/fixes/fix_dict.py": 3760, "/usr/lib/python3.8/lib2to3/fixes/fix_except.py": 3344, "/usr/lib/python3.8/lib2to3/fixes/fix_exec.py": 979, "/usr/lib/python3.8/lib2to3/fixes/fix_execfile.py": 2048, "/usr/lib/python3.8/lib2to3/fixes/fix_exitfunc.py": 2495, "/usr/lib/python3.8/lib2to3/fixes/fix_filter.py": 2765, "/usr/lib/python3.8/lib2to3/fixes/fix_funcattrs.py": 644, "/usr/lib/python3.8/lib2to3/fixes/fix_future.py": 547, "/usr/lib/python3.8/lib2to3/fixes/fix_getcwdu.py": 451, "/usr/lib/python3.8/lib2to3/fixes/fix_has_key.py": 3196, "/usr/lib/python3.8/lib2to3/fixes/fix_idioms.py": 4876, "/usr/lib/python3.8/lib2to3/fixes/fix_import.py": 3256, "/usr/lib/python3.8/lib2to3/fixes/fix_imports.py": 5684, "/usr/lib/python3.8/lib2to3/fixes/fix_imports2.py": 289, "/usr/lib/python3.8/lib2to3/fixes/fix_input.py": 708, "/usr/lib/python3.8/lib2to3/fixes/fix_intern.py": 1144, "/usr/lib/python3.8/lib2to3/fixes/fix_isinstance.py": 1608, "/usr/lib/python3.8/lib2to3/fixes/fix_itertools.py": 1548, "/usr/lib/python3.8/lib2to3/fixes/fix_itertools_imports.py": 2086, "/usr/lib/python3.8/lib2to3/fixes/fix_long.py": 476, "/usr/lib/python3.8/lib2to3/fixes/fix_map.py": 3640, "/usr/lib/python3.8/lib2to3/fixes/fix_metaclass.py": 8196, "/usr/lib/python3.8/lib2to3/fixes/fix_methodattrs.py": 606, "/usr/lib/python3.8/lib2to3/fixes/fix_ne.py": 571, "/usr/lib/python3.8/lib2to3/fixes/fix_next.py": 3174, "/usr/lib/python3.8/lib2to3/fixes/fix_nonzero.py": 591, "/usr/lib/python3.8/lib2to3/fixes/fix_numliterals.py": 768, "/usr/lib/python3.8/lib2to3/fixes/fix_operator.py": 3426, "/usr/lib/python3.8/lib2to3/fixes/fix_paren.py": 1227, "/usr/lib/python3.8/lib2to3/fixes/fix_print.py": 2844, "/usr/lib/python3.8/lib2to3/fixes/fix_raise.py": 2926, "/usr/lib/python3.8/lib2to3/fixes/fix_raw_input.py": 454, "/usr/lib/python3.8/lib2to3/fixes/fix_reduce.py": 837, "/usr/lib/python3.8/lib2to3/fixes/fix_reload.py": 1081, "/usr/lib/python3.8/lib2to3/fixes/fix_renames.py": 2221, "/usr/lib/python3.8/lib2to3/fixes/fix_repr.py": 613, "/usr/lib/python3.8/lib2to3/fixes/fix_set_literal.py": 1697, "/usr/lib/python3.8/lib2to3/fixes/fix_standarderror.py": 449, "/usr/lib/python3.8/lib2to3/fixes/fix_sys_exc.py": 1034, "/usr/lib/python3.8/lib2to3/fixes/fix_throw.py": 1582, "/usr/lib/python3.8/lib2to3/fixes/fix_tuple_params.py": 5565, "/usr/lib/python3.8/lib2to3/fixes/fix_types.py": 1774, "/usr/lib/python3.8/lib2to3/fixes/fix_unicode.py": 1256, "/usr/lib/python3.8/lib2to3/fixes/fix_urllib.py": 8353, "/usr/lib/python3.8/lib2to3/fixes/fix_ws_comma.py": 1090, "/usr/lib/python3.8/lib2to3/fixes/fix_xrange.py": 2694, "/usr/lib/python3.8/lib2to3/fixes/fix_xreadlines.py": 689, "/usr/lib/python3.8/lib2to3/fixes/fix_zip.py": 1289, "/usr/lib/python3.8/lib2to3/main.py": 11653, "/usr/lib/python3.8/lib2to3/patcomp.py": 7054, "/usr/lib/python3.8/lib2to3/pgen2/__init__.py": 143, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/__init__.cpython-38.opt-1.pyc": 165, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/__init__.cpython-38.opt-2.pyc": 137, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/__init__.cpython-38.pyc": 165, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/conv.cpython-38.opt-1.pyc": 6138, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/conv.cpython-38.opt-2.pyc": 3656, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/conv.cpython-38.pyc": 7020, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/driver.cpython-38.opt-1.pyc": 5093, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/driver.cpython-38.opt-2.pyc": 4097, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/driver.cpython-38.pyc": 5141, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/grammar.cpython-38.opt-1.pyc": 5665, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/grammar.cpython-38.opt-2.pyc": 2514, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/grammar.cpython-38.pyc": 5665, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/literals.cpython-38.opt-1.pyc": 1366, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/literals.cpython-38.opt-2.pyc": 1289, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/literals.cpython-38.pyc": 1554, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/parse.cpython-38.opt-1.pyc": 6468, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/parse.cpython-38.opt-2.pyc": 3244, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/parse.cpython-38.pyc": 6498, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/pgen.cpython-38.opt-1.pyc": 9327, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/pgen.cpython-38.opt-2.pyc": 9327, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/pgen.cpython-38.pyc": 9776, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/token.cpython-38.opt-1.pyc": 1887, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/token.cpython-38.opt-2.pyc": 1837, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/token.cpython-38.pyc": 1887, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/tokenize.cpython-38.opt-1.pyc": 15202, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/tokenize.cpython-38.opt-2.pyc": 11308, "/usr/lib/python3.8/lib2to3/pgen2/__pycache__/tokenize.cpython-38.pyc": 15272, "/usr/lib/python3.8/lib2to3/pgen2/conv.py": 9642, "/usr/lib/python3.8/lib2to3/pgen2/driver.py": 5969, "/usr/lib/python3.8/lib2to3/pgen2/grammar.py": 5533, "/usr/lib/python3.8/lib2to3/pgen2/literals.py": 1635, "/usr/lib/python3.8/lib2to3/pgen2/parse.py": 8155, "/usr/lib/python3.8/lib2to3/pgen2/pgen.py": 13812, "/usr/lib/python3.8/lib2to3/pgen2/token.py": 1302, "/usr/lib/python3.8/lib2to3/pgen2/tokenize.py": 21055, "/usr/lib/python3.8/lib2to3/pygram.py": 1305, "/usr/lib/python3.8/lib2to3/pytree.py": 27960, "/usr/lib/python3.8/lib2to3/refactor.py": 27399}
FILERDEPENDSFLIST_python3-2to3: /usr/lib/python3.8/lib2to3/pgen2/token.py /usr/bin/2to3-3.8
FILERDEPENDS_/usr/lib/python3.8/lib2to3/pgen2/token.py_python3-2to3:  /usr/bin/env
FILERDEPENDS_/usr/bin/2to3-3.8_python3-2to3:  /usr/bin/python3.8
PKGSIZE_python3-2to3: 987252
